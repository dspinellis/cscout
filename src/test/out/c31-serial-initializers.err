Processing workspace TestWS
Processing directory test/c
Entering directory test/c
Processing project Prj1
Processing file c31-serial-initializers.c
Consecutive elements in arrays
Consecutive elements in structures
Missing elements in arrays and structures
With optional comma
Without optional comma
Set current element with structure (w opt comma)
Set current element with structure
Set current element with structure element
Set current element in array
Set current element to structure element and move to array
Set current element in array element
Initialize structure with an element of that type
Solaris example
Done processing file c31-serial-initializers.c
Done processing project Prj1
Processing project Prj2
Processing file c31-serial-initializers.c
Consecutive elements in arrays
Consecutive elements in structures
Missing elements in arrays and structures
With optional comma
Without optional comma
Set current element with structure (w opt comma)
Set current element with structure
Set current element with structure element
Set current element in array
Set current element to structure element and move to array
Set current element in array element
Initialize structure with an element of that type
Solaris example
Done processing file c31-serial-initializers.c
Processing file prj2.c
Done processing file prj2.c
Done processing project Prj2
Exiting directory test/c
Done processing directory test/c
Done processing workspace TestWS
Post-processing /cscout/example/.cscout/host-defs.h
Post-processing /cscout/example/.cscout/host-incs.h
Post-processing /cscout/src/makecs.cs
Post-processing /cscout/src/test/c/c31-serial-initializers.c
Post-processing /cscout/src/test/c/prj2.c
Processing identifiers
2%4%6%8%10%13%15%17%19%21%23%26%28%30%32%34%36%39%41%43%45%47%50%52%54%56%58%60%63%65%67%69%71%73%76%78%80%82%84%86%89%91%93%95%97%100%
