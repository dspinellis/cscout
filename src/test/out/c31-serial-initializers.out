Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	FUNMACRO	MACROARG	CPPCONST	CPPSTRVAL	DEFCCONSTVAL	NOTDEFCCONSTVAL	EXPCCONSTVAL	NOTEXPCCONSTVAL	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
79	sa	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
91	MAXDIGIT	0	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
179	a1	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
219	a2	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
279	a	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	0
321	x	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
389	sb	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
421	main	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
489	b1	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
541	qqq	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
669	b2	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
819	b3	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
821	foo	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
909	b4	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1599	sb1	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2689	sb2	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
2975	__DATE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3295	__TIME__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4019	sb3	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
4055	__STDC__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
5109	sb4	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
5355	_cscout_dummy1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
6289	sb5	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
7709	sb6	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
8769	sb7	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
9899	sb8	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
10909	sb10	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
12409	sb11	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
13629	sb12	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
14369	func1	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
15489	sb13	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
16579	fs_func	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
16749	error	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
16869	fs_func_p	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	1	0	0	0	1	0	0
17139	fs_operation_def	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1
17399	name	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
17569	func	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
17649	fs_operation_def_t	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	1	0	0	0	1	0	0
17969	devinit	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	1
18089	fstype	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
18229	name	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
18629	dev_vfsops_tbl	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	421
2	535	5355
2	558	5355
4	7	79
4	17	179
4	21	219
4	27	279
4	38	389
4	48	489
4	63	79
4	66	669
4	78	79
4	81	819
4	90	909
4	156	389
4	159	1599
4	187	179
4	265	389
4	268	2689
4	313	179
4	398	389
4	401	4019
4	444	179
4	507	389
4	510	5109
4	559	179
4	625	389
4	628	6289
4	675	179
4	767	389
4	770	7709
4	780	669
4	795	179
4	873	389
4	876	8769
4	886	669
4	900	179
4	986	389
4	989	9899
4	999	669
4	1002	219
4	1014	179
4	1087	389
4	1090	10909
4	1101	489
4	1116	179
4	1135	179
4	1237	389
4	1240	12409
4	1251	819
4	1254	179
4	1359	389
4	1362	13629
4	1373	489
4	1392	179
4	1411	179
4	1436	14369
4	1545	389
4	1548	15489
4	1573	279
4	1588	179
4	1657	16579
4	1674	16749
4	1686	16869
4	1713	17139
4	1739	17399
4	1746	16869
4	1756	17569
4	1764	17649
4	1796	17969
4	1808	18089
4	1822	18229
4	1843	17649
4	1862	18629
4	1895	16749
5	8	91
5	31	321
5	41	421
5	53	541
5	81	821
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	0	struct 
4	9	 {\u0000a	int 
4	19	, 
4	23	;\u0000a} 
4	28	;\u0000a\u0000astruct 
4	40	 {\u0000a	int 
4	50	[3];\u0000a	struct 
4	65	 
4	68	;\u0000a	struct 
4	80	 
4	83	;\u0000a	int 
4	92	[3];\u0000a};\u0000a\u0000a#pragma echo 
4	148	\u0000astruct 
4	158	 
4	162	 = {\u0000a	0, 0, 0,	
4	183		{ .
4	189	 = 4},	
4	202	};\u0000a\u0000a#pragma echo 
4	257	\u0000astruct 
4	267	 
4	271	 = {\u0000a	{0, 0, 0,},	
4	295		1, 2,		
4	309		{ .
4	315	 = 4},	
4	328	};\u0000a\u0000a#pragma echo 
4	390	\u0000astruct 
4	400	 
4	404	 = {\u0000a	{0, 0,},	
4	425		{1, },		
4	440		{ .
4	446	 = 4},	
4	459	};\u0000a\u0000a#pragma echo 
4	499	\u0000astruct 
4	509	 
4	513	 = {\u0000a	{1, 2, 3,} ,	
4	538		{4, 5, },	
4	555		{ .
4	561	 = 4},	
4	574	};\u0000a\u0000a#pragma echo 
4	617	\u0000astruct 
4	627	 
4	631	 = {\u0000a	{1, 2, 3} ,	
4	655		{4, 5},		
4	671		{ .
4	677	 = 4},	
4	690	};\u0000a\u0000a#pragma echo 
4	759	\u0000astruct 
4	769	 
4	773	 = {\u0000a	.
4	782	 = {1,},\u0000a	{ .
4	797	 = 4},	
4	810	};\u0000a\u0000a#pragma echo 
4	865	\u0000astruct 
4	875	 
4	879	 = {\u0000a	.
4	888	 = {1},\u0000a	{ .
4	902	 = 4},	
4	915	};\u0000a\u0000a#pragma echo 
4	978	\u0000astruct 
4	988	 
4	992	 = {\u0000a	.
4	1001	.
4	1004	 = 1,\u0000a	{ .
4	1016	 = 4},	
4	1029	};\u0000a\u0000a\u0000a#pragma echo 
4	1079	\u0000astruct 
4	1089	 
4	1094	 = {\u0000a	.
4	1103	 = {2,},\u0000a	{ .
4	1118	 = 4},	
4	1131		{ .
4	1137	 = 4},	
4	1150	};\u0000a\u0000a#pragma echo 
4	1229	\u0000astruct 
4	1239	 
4	1244	 = {\u0000a	.
4	1253	.
4	1256	 = 4,	
4	1268		5,		
4	1279		{ [2] = 8 },\u0000a};\u0000a\u0000a\u0000a#pragma echo 
4	1351	\u0000astruct 
4	1361	 
4	1366	 = {\u0000a	.
4	1375	[1] = 2,\u0000a	3,\u0000a	{ .
4	1394	 = 4},	
4	1407		{ .
4	1413	 = 4},	
4	1426	};\u0000a\u0000a\u0000avoid\u0000a
4	1441	(void)\u0000a{\u0000a	#pragma echo 
4	1517	\u0000a	
4	1537		struct 
4	1547	 
4	1552	 = {\u0000a		{0},		
4	1571			
4	1574	,		
4	1583			{ .
4	1590	 = 4},	
4	1603		};\u0000a}\u0000a\u0000a#pragma echo 
4	1642	\u0000atypedef union 
4	1664	 {\u0000a	int (*
4	1679	)();\u0000a} 
4	1695	;\u0000a\u0000atypedef struct 
4	1729	 {\u0000a	char *
4	1743	;\u0000a	
4	1755	 
4	1760	;\u0000a} 
4	1782	;\u0000a\u0000astatic int\u0000a
4	1803	(int 
4	1814	, char *
4	1826	)\u0000a{\u0000astatic const 
4	1861	 
4	1876	[] = {\u0000a
4	1890	, { .
4	1900	 = 0 },\u0000a};\u0000a}\u0000a\u0000a
5	0	#define 
5	16	 11\u0000aextern int 
5	32	;\u0000aextern 
5	45	();\u0000aint 
5	56	;\u0000a\u0000a
5	80	\u0000a
5	84	() {\u0000a	
5	107	}\u0000a\u0000a
Table: Projects
PID	NAME
23	unspecified
24	Prj1
25	Prj2
Table: IdProj
EID	PID
79	24
179	24
219	24
279	24
389	24
421	24
489	24
669	24
819	24
909	24
1599	24
2689	24
2975	24
3295	24
3575	24
3865	24
4019	24
4055	24
5109	24
5355	24
6289	24
7709	24
8769	24
9899	24
10909	24
12409	24
13629	24
14369	24
15489	24
16579	24
16749	24
16869	24
17139	24
17399	24
17569	24
17649	24
17969	24
18089	24
18229	24
18629	24
79	25
91	25
179	25
219	25
279	25
321	25
389	25
421	25
489	25
541	25
669	25
819	25
821	25
909	25
1599	25
2689	25
2975	25
3295	25
3575	25
3865	25
4019	25
4055	25
5109	25
5355	25
6289	25
7709	25
8769	25
9899	25
10909	25
12409	25
13629	25
14369	25
15489	25
16579	25
16749	25
16869	25
17139	25
17399	25
17569	25
17649	25
17969	25
18089	25
18229	25
18629	25
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c31-serial-initializers.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	1	0	0	0	0	0	0
2	1	548	367	29	0	3	22	61	47	NULL	1	1	0	0	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	0	0	0	0	0	0	0
3	1	243	187	13	0	2	13	48	28	NULL	1	1	0	0	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	53	0	3	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	404	25	59	0	49	0	1	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	28	10	NULL	46	13	3	NULL	20	0	NULL	NULL	NULL	1	1	12	0	4	9	0	0
4	1	1743	93	354	27	0	118	67	66	NULL	3	1	0	0	0	13	0	0	0	455	25	59	0	49	0	14	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	14	5	0	23	13	3	0	20	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	7	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	NULL	1	0	2	0	0	0	0	0
5	1	95	32	17	2	0	11	20	13	NULL	1	1	0	0	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	24
3	24
4	24
1	25
2	25
3	25
4	25
5	25
Table: LineProj
FID	PID	LNUM
2	24	1
2	24	2
2	24	3
2	24	4
2	24	5
2	24	6
2	24	7
2	24	8
2	24	9
2	24	10
2	24	11
2	24	12
2	24	13
2	24	14
2	24	15
2	24	16
2	24	17
2	24	18
2	24	19
2	24	20
2	24	21
2	24	22
3	24	1
3	24	2
3	24	3
3	24	4
3	24	5
3	24	6
3	24	7
3	24	8
3	24	9
3	24	10
3	24	11
3	24	12
3	24	13
4	24	1
4	24	2
4	24	3
4	24	4
4	24	5
4	24	6
4	24	7
4	24	8
4	24	9
4	24	10
4	24	11
4	24	12
4	24	13
4	24	14
4	24	15
4	24	16
4	24	17
4	24	18
4	24	19
4	24	20
4	24	21
4	24	22
4	24	23
4	24	24
4	24	25
4	24	26
4	24	27
4	24	28
4	24	29
4	24	30
4	24	31
4	24	32
4	24	33
4	24	34
4	24	35
4	24	36
4	24	37
4	24	38
4	24	39
4	24	40
4	24	41
4	24	42
4	24	43
4	24	44
4	24	45
4	24	46
4	24	47
4	24	48
4	24	49
4	24	50
4	24	51
4	24	52
4	24	53
4	24	54
4	24	55
4	24	56
4	24	57
4	24	58
4	24	59
4	24	60
4	24	61
4	24	62
4	24	63
4	24	64
4	24	65
4	24	66
4	24	67
4	24	68
4	24	69
4	24	70
4	24	71
4	24	72
4	24	73
4	24	74
4	24	75
4	24	76
4	24	77
4	24	78
4	24	79
4	24	80
4	24	81
4	24	82
4	24	83
4	24	84
4	24	85
4	24	86
4	24	87
4	24	88
4	24	89
4	24	90
4	24	91
4	24	92
4	24	93
4	24	94
4	24	95
4	24	96
4	24	97
4	24	98
4	24	99
4	24	100
4	24	101
4	24	102
4	24	103
4	24	104
4	24	105
4	24	106
4	24	107
4	24	108
4	24	109
4	24	110
4	24	111
4	24	112
4	24	113
4	24	114
4	24	115
4	24	116
4	24	117
4	24	118
2	25	1
2	25	2
2	25	3
2	25	4
2	25	5
2	25	6
2	25	7
2	25	8
2	25	9
2	25	10
2	25	11
2	25	12
2	25	13
2	25	14
2	25	15
2	25	16
2	25	17
2	25	18
2	25	19
2	25	20
2	25	21
2	25	22
3	25	1
3	25	2
3	25	3
3	25	4
3	25	5
3	25	6
3	25	7
3	25	8
3	25	9
3	25	10
3	25	11
3	25	12
3	25	13
4	25	1
4	25	2
4	25	3
4	25	4
4	25	5
4	25	6
4	25	7
4	25	8
4	25	9
4	25	10
4	25	11
4	25	12
4	25	13
4	25	14
4	25	15
4	25	16
4	25	17
4	25	18
4	25	19
4	25	20
4	25	21
4	25	22
4	25	23
4	25	24
4	25	25
4	25	26
4	25	27
4	25	28
4	25	29
4	25	30
4	25	31
4	25	32
4	25	33
4	25	34
4	25	35
4	25	36
4	25	37
4	25	38
4	25	39
4	25	40
4	25	41
4	25	42
4	25	43
4	25	44
4	25	45
4	25	46
4	25	47
4	25	48
4	25	49
4	25	50
4	25	51
4	25	52
4	25	53
4	25	54
4	25	55
4	25	56
4	25	57
4	25	58
4	25	59
4	25	60
4	25	61
4	25	62
4	25	63
4	25	64
4	25	65
4	25	66
4	25	67
4	25	68
4	25	69
4	25	70
4	25	71
4	25	72
4	25	73
4	25	74
4	25	75
4	25	76
4	25	77
4	25	78
4	25	79
4	25	80
4	25	81
4	25	82
4	25	83
4	25	84
4	25	85
4	25	86
4	25	87
4	25	88
4	25	89
4	25	90
4	25	91
4	25	92
4	25	93
4	25	94
4	25	95
4	25	96
4	25	97
4	25	98
4	25	99
4	25	100
4	25	101
4	25	102
4	25	103
4	25	104
4	25	105
4	25	106
4	25	107
4	25	108
4	25	109
4	25	110
4	25	111
4	25	112
4	25	113
4	25	114
4	25	115
4	25	116
4	25	117
4	25	118
5	25	1
5	25	2
5	25	3
5	25	4
5	25	5
5	25	6
5	25	7
5	25	8
5	25	9
5	25	10
5	25	11
Table: LinePos
FID	FOFFSET	LNUM
2	0	1
2	4	2
2	51	3
2	54	4
2	100	5
2	119	6
2	122	7
2	184	8
2	231	9
2	234	10
2	284	11
2	288	12
2	289	13
2	321	14
2	349	15
2	378	16
2	397	17
2	416	18
2	417	19
2	467	20
2	479	21
2	523	22
3	0	1
3	4	2
3	51	3
3	54	4
3	78	5
3	97	6
3	100	7
3	149	8
3	153	9
3	154	10
3	155	11
3	196	12
3	240	13
4	0	1
4	12	2
4	25	3
4	30	4
4	31	5
4	43	6
4	55	7
4	70	8
4	85	9
4	97	10
4	100	11
4	101	12
4	149	13
4	167	14
4	183	15
4	202	16
4	205	17
4	206	18
4	258	19
4	276	20
4	295	21
4	309	22
4	328	23
4	331	24
4	332	25
4	391	26
4	409	27
4	425	28
4	440	29
4	459	30
4	462	31
4	463	32
4	500	33
4	518	34
4	538	35
4	555	36
4	574	37
4	577	38
4	578	39
4	618	40
4	636	41
4	655	42
4	671	43
4	690	44
4	693	45
4	694	46
4	760	47
4	778	48
4	791	49
4	810	50
4	813	51
4	814	52
4	866	53
4	884	54
4	896	55
4	915	56
4	918	57
4	919	58
4	979	59
4	997	60
4	1010	61
4	1029	62
4	1032	63
4	1033	64
4	1034	65
4	1080	66
4	1099	67
4	1112	68
4	1131	69
4	1150	70
4	1153	71
4	1154	72
4	1230	73
4	1249	74
4	1268	75
4	1279	76
4	1293	77
4	1296	78
4	1297	79
4	1298	80
4	1352	81
4	1371	82
4	1384	83
4	1388	84
4	1407	85
4	1426	86
4	1429	87
4	1430	88
4	1431	89
4	1436	90
4	1448	91
4	1450	92
4	1518	93
4	1537	94
4	1557	95
4	1571	96
4	1583	97
4	1603	98
4	1607	99
4	1609	100
4	1610	101
4	1643	102
4	1667	103
4	1684	104
4	1697	105
4	1698	106
4	1732	107
4	1745	108
4	1762	109
4	1784	110
4	1785	111
4	1796	112
4	1828	113
4	1830	114
4	1883	115
4	1908	116
4	1911	117
4	1913	118
5	0	1
5	20	2
5	34	3
5	49	4
5	58	5
5	59	6
5	80	7
5	81	8
5	89	9
5	107	10
5	109	11
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
25	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
24	2	2	1
24	4	3	1
24	4	4	1
25	2	2	1
25	2	2	1
25	4	3	1
25	4	4	1
25	5	3	1
25	5	5	1
Table: Providers
PID	CUID	PROVIDERID
24	2	2
24	4	4
25	2	2
25	2	2
25	4	4
25	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
25	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
821	foo	0	1	1	0	5	81	0
4715	main	0	0	1	0	2	471	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
14369	func1	0	1	1	0	4	1436	0
17969	devinit	0	1	1	1	4	1796	0
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
821	5	88	5	108
5355	2	557	2	577
14369	4	1449	4	1608
17969	4	1829	4	1912
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NGNSOC	NMPARAM	NFPARAM	NEPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
821	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	29	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
821	1	21	14	3	1	0	3	17	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	NULL	21	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
5355	1	22	0	3	0	0	1	21	5	NULL	0	1	0	0	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	0	NULL	0	NULL	0	1	1	1	1	1	1.0	0.0	1.0
14369	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	38	0	2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	21	1	3	2	2	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	2	1	0	NULL	2	0	NULL	35	NULL	0	NULL	0	0	1	1	1	0.0	18.0947	0.0
14369	1	151	24	29	4	0	9	59	54	NULL	2	0	0	0	0	1	0	0	0	25	1	3	2	2	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	2	1	0	0	2	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	18.0947	0.0
17969	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	32	0	2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	20	1	3	2	1	0	1	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	2	0	1	NULL	2	0	NULL	38	NULL	2	NULL	0	0	1	1	1	0.0	13.9316	0.0
17969	1	78	0	15	0	0	5	45	32	NULL	2	0	0	0	0	0	0	0	0	20	1	3	2	1	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	2	0	1	0	2	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	13.9316	0.0
Table: FunctionId
FUNCTIONID	ORDINAL	EID
821	0	821
4715	0	421
5355	0	5355
14369	0	14369
17969	0	17969
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
